README.md

```diff
@@ -27,11 +27,113 @@ The documentation of the 121 platform can be found on the Wiki of this repositor
 ## Getting Started
 To set up a local development-environment:
 
-- Verify that your environment meets the requirements defined in:
-  - [interfaces/README](interfaces/README.md#environment-requirements) for the front-end interfaces
-  - [services/README](services/README.md) for the back-end services
-- Run: `npm run install:all` from *this* folder
-- Run: `npm run start:all` from *this* folder
+
+Guide to setup local development environment for 121.
+
+The repository as can be seen contains, services and interfaces directories. The services are run using docker environment and the interfaces are individual Angular applications.
+
+# 1. Windows OS
+## Install dependencies
+### Install Node.js (https://nodejs.org/en/download/)
+#### Choose an appropriate executable to install node.js
+### Install Git (https://git-scm.com/download/win)
+### Install Docker (https://docs.docker.com/docker-for-windows/install/)
+
+
+# 2. Linux
+## Install dependencies
+### Node.JS
+
+        sudo apt install nodejs
+
+### Install Git
+
+        sudo apt install git-all
+
+### Install Docker
+
+On linux distributions we need to install docker enginer and docker compose respectively. On other platforms docker-compose enginer is available through Docker Desktop. Read more at: https://docs.docker.com/engine/install/
+
+Docker compose relies on docker engine, so in order to get started we must install docker-engine first. (Read more at: https://docs.docker.com/engine/install/ubuntu/). Theere are various ways of installing docker engine (repository, packages or automated scripts). The official recommended way doing that is through setting up a repository.
+
+#### First off, uninstall any old version of docker-engine.
+
+    sudo apt-get remove docker docker-engine docker.io containerd runc
+
+#### Choose an installation method from the provided list at https://docs.docker.com/engine/install/ubuntu/#installation-methods and install docker-engine
```

Comment:
It says to install docker-engine, but when opening the link and scrolling to "Install Docker Engine" that does not work. Installing docker is probably needed as well.